photon-api/src/apis/default_api.rs

```diff
@@ -956,17 +991,23 @@ pub async fn get_multiple_compressed_account_proofs_post(
 
     let local_var_client = &local_var_configuration.client;
 
-    let local_var_uri_str = format!(
+    let mut local_var_uri_str = format!(
         "{}/getMultipleCompressedAccountProofs",
         local_var_configuration.base_path
     );
+
+    if let Some(ref api_key) = local_var_configuration.api_key {
+        local_var_uri_str = format!("{}?api_key={}", local_var_uri_str, api_key.key);
+    }
+
```

Comment:
can we use `append_api_key` here as well?

Reply:
sure, thanks!